project-metrics/metrics_service/metrics/base.py

```diff
@@ -32,29 +40,81 @@ def label(self) -> Text:
   @property
   def formatted_result(self) -> Text:
     """The formatted result value (ex. 3w, 80%, 100PRs)."""
-    return self._format_value(self._result.value) if self._result else '?'
-
-  @property
-  def result(self) -> Optional[models.MetricResult]:
-    """The result of a computation of the metric."""
-    return self._result
+    return self._format_value(self.result.value) if self.result else '?'
```

Comment:
This wouldn't (or at least, shouldn't) pass PyType checks because `self.result` is not a defined attribute of this class (it's not defined in `__init__`) - there is a reverse expectation of inheritance here. I'm not sure what the best solution here is. Maybe don't split the classes at all - there's no reason to create a deep chain of inheriting classes, the codebase doesn't benefit from it

Reply:
Done.